docs/quickstart/server.mdx

```diff
@@ -660,7 +660,7 @@ Finally, implement the main function to run the server:
 async function main() {
   const transport = new StdioServerTransport();
   await server.connect(transport);
-  console.error("Weather MCP Server running on stdio");
+  console.info("Weather MCP Server running on stdio");
```

Comment:
This won't work - `console.info` writes to stdio, where the MCP server listens to. Using `console.error` is intentional.